windows_containers/index.adoc

```diff
@@ -6,6 +6,10 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
+Documentation for {productwinc} will be available for {product-title} {product-version} in the near future.
```

Comment:
Good question. I recall doing an informal WMCO team review when I started doing this for WMCO 10.16. Simply because the docs will absolutely be available in the near future.  @listey WDYT?